File: Lab 01/Lab Exercise 1 - Hello World/C++/program.cpp
#include "message.h"

int main() {
	message *msg1 = new message("Hello World");
	message *msg2 = new message("This is a C++ program");
	message *msg3 = new message("With 3 Message objects");

	msg1->print();
	msg2->print();
	msg3->print();
	return 0;
}

File: Lab 01/Lab Exercise 1 - Hello World/C#/Program.cs
class Program {
	public static void Main() {
		Message msg1 = new Message("Hello World");
		Message msg2 = new Message("This is a C# program");
		Message msg3 = new Message("With 3 Message objects");

		msg1.Print();
		msg2.Print();
		msg3.Print();
	}
}